tools/generate-examples.rb

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "fileutils"
+require "pathname"
+
+VERBOSE = false
+EXAMPLES_DIR = File.expand_path("#{__dir__}/../examples", __dir__)
```

Comment:
I'd write this as 
```suggestion
EXAMPLES_DIR = File.expand_path("../examples", __dir__)
```
As is, I think the second arg will be unused